servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java

```diff
@@ -335,4 +335,32 @@ public void abortCommitedBlockTest() throws Exception {
     mBlockStore.commitBlock(USER_ID1, TEMP_BLOCK_ID);
     mBlockStore.abortBlock(USER_ID1, TEMP_BLOCK_ID);
   }
+
+  @Test
+  public void commitBlockTwice() throws Exception {
```

Comment:
`commitBlockTwice` -> `commitBlockTwiceTest`
This applies to other testcase name too